test/SourceKit/CursorInfo/cursor_info.swift

```diff
@@ -804,8 +814,47 @@ func checkAnyIsAKeyword(x: Any) {}
 // CHECK93-NEXT: <decl.enumelement><syntaxtype.keyword>case</syntaxtype.keyword> <decl.name>b</decl.name> = <syntaxtype.string>&quot;f&quot;</syntaxtype.string></decl.enumelement>
 
 // RUN: %sourcekitd-test -req=cursor -pos=227:14 %s -- -F %S/../Inputs/libIDE-mock-sdk -I %t.tmp %s | %FileCheck -check-prefix=CHECK94 %s
-// CHECK94: <empty cursor info; internal diagnostic: "Resolved to incomplete expression or statement.">
+// CHECK94: source.lang.swift.ref.function.constructor
+// CHECK94-NEXT: init(_builtinStringLiteral:utf8CodeUnitCount:isASCII:)
```

Comment:
I just noticed that this shows an initializer that’s hidden from all swift interfaces (because it starts with an underscore) and takes a `Builtin` type, which is not accessible to users. What I would suggest is that instead of showing the builtin initializers, we show the type of the literal instead. See my other comment of how I think that should be done.

For other types that use an initializer that’s not a builtin initializer, I think we should still refer to the initializer.

Reply:
Ok, fixed. I did the same for the object literals as their inits also seem to be undescored.